modules/data.atmosphere/R/met.process.R

```diff
@@ -339,8 +481,12 @@ met.process <- function(site, input_met, start_date, end_date, model, host, dbpa
     fcn       <- paste0("met2model.",model)
     lst       <- site.lst(site,con)
 
+    for (dates in 1: length(input.info$dates$download$start_date)){
```

Comment:
Somewhere here in this chunk of model-specific code you need to be checking the model registry to see if this is an update or not, and then below you should be creating a new input record for some models vs. updating the existing input record for others